docs/operating-eck/troubleshooting/common-problems.asciidoc

```diff
@@ -31,3 +31,56 @@ Error from server (Timeout): error when creating "elasticsearch.yaml": Timeout:
 
 
 This error is usually an indication of a problem communicating with the validating webhook. If you are running ECK on a private Google Kubernetes Engine (GKE) cluster, you may need to link:https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters#add_firewall_rules[add a firewall rule] whitelisting port 9443 from the API server. Another possible cause for failure is if a strict <<{p}-webhook-network-policies,network policy>> is in effect. Refer to the <<{p}-webhook-troubleshooting,webhook troubleshooting>> documentation for  more details and workarounds.
+
+[id="{p}-{page_id}-owner-refs"]
+== Copying secrets with Owner References
+
+Copying the Elasticsearch Secrets generated by ECK (for instance, the certificate authority or the elastic user) into another namespace wholesale can trigger a link:https://github.com/kubernetes/kubernetes/issues/65200[catastrophic Kubernetes bug] which can delete all of the Elasticsearch-related resources (for instance, the data volumes). To avoid this, when copying Secrets between namespaces, be sure to remove the `metadata.ownerReferences` section. For example, a source secret might be:
```

Comment:
```suggestion
Copying the Elasticsearch Secrets generated by ECK (for instance, the certificate authority or the elastic user) into another namespace wholesale can trigger a link:https://github.com/kubernetes/kubernetes/issues/65200[Kubernetes bug] which can delete all of the Elasticsearch-related resources (for instance, the data volumes). To avoid this, when copying Secrets between namespaces, be sure to remove the `metadata.ownerReferences` section. For example, a source secret might be:
```